vnext/Microsoft.ReactNative.sln

```diff
@@ -138,20 +138,19 @@ Global
 		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{da8b35b3-da00-4b02-bde6-6a397b3fd46b}*SharedItemsImports = 9
 		include\Include.vcxitems*{ef074ba1-2d54-4d49-a28e-5e040b47cd2e}*SharedItemsImports = 9
 		Chakra\Chakra.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
-		JSI\Shared\JSI.Shared.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
```

Comment:
>Shared [](start = 17, length = 6)

You might want to check with @Julio 's change, he has a PR to remove JSI.Shared

Reply:
Yes, we have discussed it with him yesterday: I have moved the new unit tests to Microsoft.ReactNative.ComponentTests. He will move the other files if I merge before his PR.